cpu/cc26x0/periph/rfc.c

```diff
@@ -125,7 +125,7 @@ void rfc_beacon(void)
         printf("%.2x", ((uint8_t *) rop)[i]);
     printf("\n");
 
-    //run_command_ptr(&rop->op);
+    run_command_ptr(&rop->op);*/
```

Comment:
again, just drop the code or replace it with code that uses your new function.
doesn't matter which because bisect wont find any working BLE commits before this change anyway ;-)